Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs

```diff
@@ -9,7 +10,7 @@ public abstract class AssertDiagnosticVerifier : DiagnosticVerifier
 	{
 		private readonly AssertCodeHelper _helper = new();
 
-		protected void VerifyError(string methodBody, string expectedDiagnosticId)
+		protected async Task VerifyErrorAsync(string methodBody, string expectedDiagnosticId)
```

Comment:
This method has the Async postfix, please remove it

Reply:
done